src/Resources/config/seo_block.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <parameter key="sonata.user.block.breadcrumb_index.class">Sonata\UserBundle\Block\Breadcrumb\UserIndexBreadcrumbBlockService</parameter>
+        <parameter key="sonata.user.block.breadcrumb_profile.class">Sonata\UserBundle\Block\Breadcrumb\UserProfileBreadcrumbBlockService</parameter>
+    </parameters>
```

Comment:
I think we no longer allow that kind of thing, it would be great if another maintainer could confirm this.

Reply:
All blocks is final now. It logic to not use parameters.